docs/meshstack.aws.sso-setup.md

```diff
@@ -0,0 +1,79 @@
+---
+id: meshstack.aws.sso-setup
+title: SSO Setup
+---
+## Configure AAD as IdP for AWS SSO
+
+This section describes how to set up AWS SSO with Azure Active Directory.
+
+### Create an Enterprise Registration
+
+Search for `aws` on Azure and choose `AWS Single Sign-on` as the enterprise registration type
```

Comment:
Here the context is missing where this enterprise registration type can be found in azure? Can we add one more hint on how to get to that screen that is shown in the screenshot?